composer.json

```diff
@@ -16,7 +16,8 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=7.4",
-        "commongateway/corebundle": ">=1.0.48"
+        "commongateway/corebundle": ">=1.0.48",
```

Comment:
We could/should consider to upgrade core to the latest version at some point but we can also postpone it forever of course 😝

Reply:
Well, in this case: I'll update, because it requires a more recent version of core, but as long as a bundle works in an old version, updating it is not necessary

Reply:
true, except that we might have done some fixes that do affect everything including your current project. Especially if you are like 50 versions behind. I would say if the bundle works with the newest (stable) version updating is necessary.